### ISSUE TITLE
should there be a generic pseudo selector for carousel buttons? 

### ISSUE THREAD

**Issue by argyleink** [flackr/carousel#12]
Currently there's left, right, top, and bottom pseudo selectors but not one that can style all of them at once. Should there be?

**Comment by rol4nd909** [comment]
That would make it a lot easier for sure so +1 ;)

**Comment by flackr** [comment]
Yes, there probably should be. I have many other ideas around how scroll buttons might work, e.g. maybe it should have a function parameter `::scroll-button(right)` instead and allow for `::scroll-button(*)` or maybe just `::scroll-button` to match all of them.
